Packages/com.unity.inputsystem/Tests/InputSystem/CoreTests_Actions.cs

```diff
@@ -2584,6 +2584,33 @@ public void Actions_CanAddProcessorsToActions()
         Assert.That(receivedVector.Value.y, Is.EqualTo(0.5678).Within(0.00001));
     }
 
+    [Test]
+    [Category("Actions")]
+    public void Actions_IncompatibleProcessorIsIgnored()
```

Comment:
Just wondering, is this actually desirable behavior? If there's a processor that doesn't apply, isn't that a problem in the setup that should be diagnosed?

Reply:
I've been thinking about the same, and considered just making it generate more meaningful errors instead. But I chose this solution instead. Some reasons:

-In the case reported in the error where we have a dpad composite binding, the processor seems to be applied on both the composite elements (where it makes sense), and the result (where it does not). Ignoring it where it does not does the right thing here, which throwing errors would not solve.

-I think  it may be convenient to not invalidate everything when changing bindings. This way, you can quickly switch types, try out different things, without having to change all the processors. Instead I chose to mark incompatible processors as "(ignored)" in the UI.

Reply:
>the processor seems to be applied on both the composite elements (where it makes sense)

Oh, that one would be a bug. The composite and its parts are each individual bindings and the processors applied to one should not affect the other.

>I think it may be convenient to not invalidate everything when changing bindings. This way, you can quickly switch types, try out different things, without having to change all the processors.

Good point I think.